src/static-landing/css/style.css

```diff
@@ -3,13 +3,13 @@ body {
 }
 
 a:link {
-  color: #fff;
-  text-decoration: none;
+  color: #036;
+  text-decoration: underline;
```

Comment:
Nit: Not necessary, underline is the default

Reply:
Fixed in https://github.com/bcgov/bcparks.ca/pull/249/commits/400e913265603d214b0214e25f0528652893be48